docs/core/resilience/index.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Introduction to resilience in .NET
+description: Learn about resilience in .NET and how to build a resilience pipeline.
+author: IEvangelist
+ms.author: dapine
+ms.date: 09/28/2023
+---
+
+# Introduction to resilience in .NET
+
+Resiliency is the ability of an app to recover from failures and continue to function. In the context of .NET programming, resilience is achieved by designing apps that can handle failures gracefully and recover quickly. To help build resilient apps in .NET, the following two packages are available on NuGet:
+
+- [Microsoft.Extensions.Resilience](https://www.nuget.org/packages/Microsoft.Extensions.Resilience): This NuGet package provides mechanisms to harden apps against transient failures.
+- [Microsoft.Extensions.Http.Resilience](https://www.nuget.org/packages/Microsoft.Extensions.Http.Resilience): This NuGet package provides resilience mechanisms specifically for the <xref:System.Net.Http.HttpClient>.
+
+These two NuGet packages are built on top of [Polly](https://github.com/dotnet/docs/pull/37270), which is a very popular open-source project. Polly is a .NET resilience and transient fault-handling library that allows developers to express strategies such as Retry, Circuit Breaker, Timeout, Bulkhead Isolation, Rate-limiting and Fallback in a fluent and thread-safe manner.
+
+> [!IMPORTANT]
+> The [Microsoft.Extensions.Http.Polly](https://www.nuget.org/packages/Microsoft.Extensions.Http.Polly) NuGet package is deprecated. Use either of the aforementioned packages instead.
+
+## Get started
+
+To get started with resilience in .NET, install the [Microsoft.Extensions.Resilience](https://www.nuget.org/packages/Microsoft.Extensions.Resilience) NuGet package.
+
+### [.NET CLI](#tab/dotnet-cli)
+
+```dotnetcli
+dotnet add package Microsoft.Extensions.Resilience --version 8.0.0
+```
+
+### [PackageReference](#tab/package-reference)
+
+```xml
+<PackageReference Include="Microsoft.Extensions.Resilience" Version="8.0.0" />
+```
+
+---
+
+For more information, see [dotnet add package](../tools/dotnet-add-package.md) or [Manage package dependencies in .NET applications](../tools/dependencies.md).
+
+## Build a resilience pipeline
+
+To use resilience, you must first build a pipeline of resilience-based strategies. Each configured strategy executes in order of configuration The entry point is an extension method on the <xref:Microsoft.Extensions.DependencyInjection.IServiceCollection> type, named `AddResiliencePipeline`. This method takes a delegate that configures the pipeline. The delegate is passed an instance of `ResiliencePipelineBuilder`, which is used to add resilience strategies to the pipeline.
```

Comment:
```suggestion
To use resilience, you must first build a pipeline of resilience-based strategies. Each configured strategy executes in order of configuration. The entry point is an extension method on the <xref:Microsoft.Extensions.DependencyInjection.IServiceCollection> type, named `AddResiliencePipeline`. This method takes an identifier of the pipeline and delegate that configures the pipeline. The delegate is passed an instance of `ResiliencePipelineBuilder`, which is used to add resilience strategies to the pipeline.
```

There is also alternative overload that has a form of `(builder, context)` for more advanced scenarios. Might note it exists.